roles/influxdb/tasks/backup.yml

```
@@ -0,0 +1,26 @@
+---
+- name : Read root token
```

```suggestion
- name: Read root token
```
Just a minor typo. :-)